.github/workflows/kind-e2e.yaml

```diff
@@ -2,7 +2,7 @@ name: e2e
 
```

Comment:
Is there any change in this kind-e2e.yaml? If it is only format change, was there any problem for the previous format?

Reply:
Just what my linter catched but I can revert

The only important change here is the ko-build action change